pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: browser_launcher
 description: A package that provides a standardized way to launch web browsers.
```

Comment:
Tweak the description – package is redundant.

`Provides a standardized way to launch web browsers for testing and tools.`?